controler/usersControler.js

```diff
@@ -0,0 +1,31 @@
+const peopleModel = require("../models/People");
+const getUsers = (req, res, next) => {
+  res.render("users");
+};
+
+const createUser = async (req, res, next) => {
+  try {
+    const peopleInfo = await peopleModel.find();
+    peopleInfo.map((data) => {
```

Comment:
Not a good approach here finding user existance from your database. Here you are finding an user existance. I am quite sure some of your columns are unique (email). Better try to find out that specific user with your unique key and your returned result should be in an object not an array. Currently you are just fetching all users and try to match your requested email with your all users email
pseudocode:
```
matchedUser = await user.findOne({ email: req.email} )
if(matchedUser) return "user already exists"
// create user if not found in database
```